lms/djangoapps/program_enrollments/rest_api/v1/utils.py

```diff
@@ -1,22 +1,20 @@
 # -*- coding: utf-8 -*-
 """
-ProgramEnrollment internal api
```

Comment:
hi @kdmccormick , the enterprise enrollment api is depending on these functions being exposed via the api.py file according to this ADR: https://github.com/edx/edx-platform/blob/master/docs/decisions/0002-inter-app-apis.rst

this file name change has resulted in breaking our functionality. there are a couple layers of weirdness here that resulted in this situation, but do you object in the mean time if i reintroduce the api.py file with functions wrapping the ones here?

i don't know how much future development is planned in this area, but it seems that these functions are not specific to the domain of program enrollments, and should be moved somewhere more appropriate since both programs and enterprise portal enrollment functionality depend on these functions. let me know what you think.

fyi @christopappas

Reply:
Hi @brittneyexline. The purpose of this PR is to work towards introducing a Python API as described in that ADR. This `api.py` file didn't meet the requirements of the ADR, and the "internal utilities" wording in the docstring made me think it didn't have any external dependents, hence me renaming it.

For the time being I have no qualms with an `api.py` here that is essentially:
```python
"""
docstring about why enterprise API needs this module to exist,
and link to ticket that will remove it
"""
from .utils import (
    # all the functions you need
)
```

In terms of development in this area, I'm currently in the process of tidying up the program_enrollments app. [This PR](https://github.com/edx/edx-platform/pull/21591) will start introducing a program_enrollments Python API that can be depended upon. Like you said, I agree that the functions that were in `api.py` and are now in `utils.py` belong somewhere else. Maybe the `courseware` app?

Reply:
hi @kdmccormick - some of the weirdness i was referring to is the fact that we were importing these functions from edx-enterprise, so the dependencies were not readily visible working only in edx-platform. in the mean time, i reintroduced the imports we were depending on (https://github.com/edx/edx-platform/pull/21593/files#diff-26734a293ffb45f242140771adde9de7R3) - if you could preserve those for the time being, we would appreciate it!

i have filed a ticket for us to follow up on moving these utility functions somewhere else: https://openedx.atlassian.net/browse/ENT-2294. we should be able to address this in the next 2-3 weeks - if it becomes more urgent let us know and we can reprioritize.

Reply:
@brittneyexline Cool, can you look at https://github.com/edx/edx-platform/pull/21605/files? Thanks for the ticket!